controller/src/main/resources/xsd/measurement.xsd

```diff
@@ -0,0 +1,64 @@
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           targetNamespace="http://uni.miskolc.ips.ilona.measurement/measurement"
+           elementFormDefault="qualified"
+           xmlns:pos="http://uni.miskolc.ips.ilona.measurement/position">
+
+    <xs:import namespace="http://uni.miskolc.ips.ilona.measurement/position" schemaLocation="position.xsd"/>
+
+
+    <xs:complexType name="MeasurementDTO">
+        <xs:sequence>
+            <xs:element name="id" type="xs:string"/>
+            <xs:element name="timestamp" type="xs:dateTime"/>
+            <xs:element name="position" type="pos:PositionDTO"/>
+            <xs:element name="wifiRSSI" minOccurs="0" maxOccurs="1">
+                <xs:complexType>
+                    <xs:sequence>
+                        <xs:element name="ap" minOccurs="1" maxOccurs="unbounded">
```

Comment:
Why is the minOccurs 1?

Reply:
wifiRSSI, Magnetometer, etc. are the elements of the measurement and they are optional. 

ap is an element of the wifiRSSI tag and it has to contains at least one ap.